src/overlays/actors/ovl_En_Toto/z_en_toto.c

```diff
@@ -702,14 +702,14 @@ void EnToto_Update(Actor* thisx, PlayState* play) {
 }
 
 void EnToto_Draw(Actor* thisx, PlayState* play) {
-    TexturePtr sp4C[] = { object_zm_Tex_008AE8, object_zm_Tex_00A068, object_zm_Tex_00A468 };
+    TexturePtr eyeTextures[] = { gTotoEyesOpenTex, gTotoEyesHalfTex, gTotoEyesClosedTex };
     EnToto* this = (EnToto*)thisx;
     s32 pad;
 
     OPEN_DISPS(play->state.gfxCtx);
 
     Gfx_SetupDL25_Opa(play->state.gfxCtx);
-    gSPSegment(POLY_OPA_DISP++, 0x08, Lib_SegmentedToVirtual(sp4C[this->blinkInfo.eyeTexIndex]));
+    gSPSegment(POLY_OPA_DISP++, 0x08, Lib_SegmentedToVirtual(eyeTextures[this->faceChange.face]));
```

Comment:
It’s worth noting that toto for some reason uses the face change system and `FaceChange_UpdateBlinkingAlt` to handle it’s blinking. The only time it’s used outside of player. So I’m not sure if that should affect the name of `FaceChange`.

Reply:
I don't think so. Name seems fine.